reg_tests/grid_gen/c96.uniform.sh

```diff
@@ -13,6 +13,8 @@ export out_dir=${WORK_DIR}/c96.uniform
 
 export res=96
 export gtype=uniform
+export add_lake=true
+export lake_data_srce=MODISP_GLDBV3
```

Comment:
add line after this:
export gridcell_lake = 1

Reply:
do we need binary_lake here?

Reply:
Yes. Good catch, @barlage